test/lib/resources/constants.go

```diff
@@ -24,9 +24,9 @@ import (
 // API versions for the resources.
 const (
 	CoreAPIVersion        = "v1"
-	EventingAPIVersion    = "eventing.knative.dev/v1beta1"
-	MessagingAPIVersion   = "messaging.knative.dev/v1beta1"
-	FlowsAPIVersion       = "flows.knative.dev/v1beta1"
+	EventingAPIVersion    = "eventing.knative.dev/v1"
+	MessagingAPIVersion   = "messaging.knative.dev/v1"
```

Comment:
/cc @cardil here is the case, I am referencing to from my review on the kafka channel upgreade tests:

https://github.com/knative-sandbox/eventing-kafka/pull/467/files#r605554669

Reply:
Faster than I thought